client/src/components/Button/Button.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Button } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import './Button.scss';
+
+export const GreenButton = (props) => {
+  const Green = styled(Button)({
```

Comment:
I wouldn't consider it blocking but I thought we were moving off of styled components lol